cypress/e2e/collective-members.spec.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @copyright Copyright (c) 2022 Jonas <jonas@freesources.org>
+ *
+ * @author Jonas <jonas@freesources.org>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ *  Tests for Collectives settings.
+ */
+
+describe('Collective members', function() {
+	before(function() {
+		cy.login('bob')
+		cy.deleteCollective('Members Collective')
+		cy.deleteAndSeedCollective('Members Collective')
+	})
+
+	beforeEach(function() {
+		cy.login('bob')
+
+		cy.get('.collectives_list_item')
+			.contains('li', 'Members Collective')
+			.find('.action-item__menutoggle')
+			.click({ force: true })
+		cy.get('button.action-button')
+			.contains('Manage members')
+			.click()
+	})
+
+	describe('Manage members', function() {
+		it('Allows to add members', function() {
+			// const addMembers = ['alice', 'Bobs Group']
```

Comment:
Any reason to leave this around? Shall we have a separate test for the groups?

Reply:
Yes, I want to switch back to test with a group as well as soon as I resolved the following problem: for now, groups that are part of members already are not filtered out of the search results. The problem is that groups have type `16` (`TYPE_GROUP`) in circle response.